tooling/bundler/src/bundle/settings.rs

```diff
@@ -156,8 +155,10 @@ pub struct PackageSettings {
 /// The updater settings.
 #[derive(Debug, Default, Clone)]
 pub struct UpdaterSettings {
+  /// Should generate v1 compatible unzipped updater
+  pub v1_compatible: bool,
   /// Signature public key.
-  pub pubkey: String,
+  pub pubkey: Option<String>,
```

Comment:
let's just remove the option, it is not needed anymore, it only existed to check whether updater was enabled or not